packages/medusa/src/loaders/helpers/routing/__tests__/index.spec.ts

```diff
@@ -0,0 +1,151 @@
+import http from "http"
+import { resolve } from "path"
+import request from "supertest"
+import express from "express"
+import { RoutesLoader } from "../index"
+import {
+  customersCreateMiddlewareMock,
+  customersGlobalMiddlewareMock,
+  storeCorsMiddlewareMock,
+} from "../__fixtures__/mocks"
+
+const mockConfigModule = {
+  projectConfig: {
+    store_cors: "http://localhost:8000",
+    admin_cors: "http://localhost:7001",
+    database_logging: false,
+  },
+  featureFlags: {},
+  plugins: [],
+}
+
+describe("RoutesLoader", function () {
+  afterEach(function () {
+    jest.clearAllMocks()
+  })
+
+  describe("Routes", function () {
+    const app = express()
+    const server = http.createServer(app)
+
+    beforeAll(async function () {
+      const rootDir = resolve(__dirname, "../__fixtures__/routers")
+
+      await new RoutesLoader({
+        app,
+        rootDir,
+        configModule: mockConfigModule,
+      }).load()
+    })
+
+    it("should return a status 200 on GET admin/order/:id", async function () {
+      await request(server)
+        .get("/admin/orders/1000")
+        .expect(200)
+        .expect("hello world")
+    })
+
+    it("should return a status 200 on POST admin/order/:id", async function () {
+      await request(server)
+        .post("/admin/orders/1000")
+        .expect(200)
+        .expect("hello world")
+    })
+
+    it("should call a multi parameters route and provide the parameters to the handler", async function () {
```

Comment:
suggestion: should we align the description such as `should call GET /customers/[customer_id]/orders/[order_id]